README.md

```diff
@@ -122,3 +127,24 @@ For instance, in a protocol that explains how to visually estimate plant cover u
 
 The subject-matter specialist will then give sensible default values for these parameters, but these values can be changed in project-specific protocols.
 
+
+## Folder and filename syntaxis
+
+In this section, we describe the conventions we adhere to for naming of files and folders.
+Each protocol will be placed in a subfolder of the folder corresponding to the theme (for thematic protocols) or project (for project-specific protocols) to which it belongs (see [Repository structure](#repository-structure)). 
+The naming syntaxis for the subfolder is `protocol-code_short-title_language-code` and will be automatically generated from input you provide to [protocolhelper functions](#starting-a-new-protocol-with-the-aid-of-protocolhelper-functions).
+
+Note that this subfolder name is stable for different versions of the same protocol, because the files in these subfolders are subject to `Git` version control. 
```

Comment:
```suggestion
Note that the names of this subfolder and of its files are stable for different versions of the same protocol, because the files in these subfolders are subject to `Git` version control and the version number is kept inside the special file `index.Rmd`. 
```

This has some relation with https://github.com/inbo/protocolhelper/pull/9#discussion_r435954290 as well.